src/viewer.js

```diff
@@ -1,6 +1,7 @@
 import Office from './view/Office'
+import { getCapabilities } from '@nextcloud/capabilities'
 
-const supportedMimes = OC.getCapabilities().officeonline.mimetypes.concat(OC.getCapabilities().officeonline.mimetypesNoDefaultOpen)
+const supportedMimes = OC.getCapabilities().officeonline.mimetypes
```

Comment:
```suggestion
const supportedMimes = getCapabilities().officeonline.mimetypes
```
With this you actually make use of the import instead of using the global function

Reply:
Okay sounds good. Thank you for the correction.